backend/tests/Functional/Controller/AdminMetadataApiAccessTest.php

```
@@ -0,0 +1,101 @@
+        self::assertTrue($this->reload($target)->isMetadataApiEnabled());
+    }
+
+    /** Otherwise a user could simply grant it back to themselves. */
+    public function testAUserCannotGrantThemselvesApiAccess(): void
+    {
+        $user = UserFactory::createOne(['metadataApiEnabled' => false])->object();
+        $this->loginAs($user);
+
+        $this->patchJson(sprintf('/api/users/%d', $user->getId()), ['metadataApiEnabled' => true]);
+
+        self::assertFalse($this->reload($user)->isMetadataApiEnabled());
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Functional\Controller\AdminMetadataApiAccessTest::assertFalse().


(staticMethod.notFound)